tests/test_orders.py

```diff
@@ -75,3 +75,74 @@ def test_create_order_refund(client, response):
     order = client.orders.get(ORDER_ID)
     refund = order.create_refund(data)
     assert isinstance(refund, Refund)
+
+
+def test_create_order(client, response):
+    """Create an order"""
+    response.post('https://api.mollie.com/v2/orders', 'order_single')
+    data = {
+        'amount': {
+            'value': '1027.99',
+            'currency': 'EUR'
+        },
+        'billingAddress': {
+            'streetAndNumber': 'Keizersgracht 313',
+            'city': 'Amsterdam',
+            'region': 'Noord-Holland',
+            'postalCode': '1234AB',
+            'country': 'NL',
+            'givenName': 'Piet',
+            'familyName': 'Mondriaan',
+            'email': 'piet@mondriaan.com',
+        },
+        'shippingAddress': {
+            'streetAndNumber': 'Prinsengracht 313',
+            'city': 'Haarlem',
+            'region': 'Noord-Holland',
+            'postalCode': '5678AB',
+            'country': 'NL',
+            'givenName': 'Chuck',
+            'familyName': 'Norris',
+            'email': 'norris@chucknorrisfacts.net'
+        },
+        'metadata': {
+            'order_id': '1337',
+            'description': 'Lego cars'
+        },
+        'consumerDateOfBirth': '1958-01-31',
+        'locale': 'nl_NL',
+        'orderNumber': '1337',
+        'redirectUrl': 'https://example.org/redirect',
+        'webhookUrl': 'https://example.org/webhook',
+        'method': 'klarnapaylater',
```

Comment:
Add to TODO list: support this new payment method in `mollie/api/objects/method.py`.